backend/apps/organizations/migrations/0034_auto_20221114_1854.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.16 on 2022-11-14 17:54
```

Comment:
Nitpicking here, so it's up to you whether you want to address this or not. This migration should ideally be in somewhat obscurely named `apps/permissions` as it only pertains to changes in groups.